code/game/jobs/job/civilian_chaplain.dm

```diff
@@ -6,304 +7,438 @@
-					B.name = "The Book of Leinster"
-					J = "Druid"
-				if("atheism")
-					B.name ="The God Delusion"
-					H.equip_or_collect(new /obj/item/clothing/head/fedora(H), slot_head)
-				if("scientology")
-					B.name = pick("The Biography of L. Ron Hubbard", "Dianetics")
-					J = "OT III"
-				if("discordianism")
-					B.name = "The Principia Discordia"
-					J = "Episkopos"
-				if("rastafarianism", "rastafari movement")
-					B.name = "The Holy Piby"
-				if("pastafarianism")
-					B.name = "The Gospel of the Flying Spaghetti Monster"
-				if("chaos")
-					B.name = pick("The Book of Lorgar", "The Book of Magnus")
-					J = "Apostate Preacher"
-				if("imperium", "imperial cult")
-					B.name = pick("An Uplifting Primer", "Codex Astartes", "Codex Hereticus")
-					if (H.gender == FEMALE)
-						J = "Prioress"
-					else
-						J = "Confessor"
-				if("toolboxia")
-					B.name = "The Toolbox Manifesto"
-				if("homosexuality")
-					B.name = pick("Guys Gone Wild", "Hunk Rump")
-				if("lol", "wtf", "gay", "penis", "ass", "poo", "badmin", "shitmin", "deadmin", "cock", "cocks", "nigger", "faggot", "dickbutt")
-					B.name = pick("Woodys Got Wood: The Aftermath", "War of the Cocks", "Sweet Bro and Hella Jef: Expanded Edition", "The Book of Pomf")
-					H.setBrainLoss(100) // starts off retarded as fuck
-				if("science")
-					B.name = pick("Principle of Relativity", "Quantum Enigma: Physics Encounters Consciousness", "Programming the Universe", "Quantum Physics and Theology", "For I Have Tasted The Fruit", "Non-Linear Genetics", "The Mysteries of Bluespace", "Playing God: Collector's Edition")
-					J = "Academician"
-				if("tribunal", "almsivi")
-					B.name = "The 36 Lessons of Vivec"
-					J = "Curate"
-				if("nine divines", "eight divines")
-					B.name = "The Elder Scrolls"
-					J = "Disciple of the Nine"
-				if("daedra")
-					B.name = pick("The Blessings of Sheogorath", "Boethiah's Pillow Book", "Invocation of Azura")
-				if("bokononism")
-					B.name = pick("The Book of Bokonon")
-				if("faith of the seven")
-					B.name = "The Seven-Pointed Star"
-					if (H.gender == FEMALE)
-						J = "Septa"
-					else
-						J = "Septon"
-				if("goa'uld")
-					B.name = "The Abydos Cartouche"
-					J = "First Prime"
-				if("unitology")
-					B.name = "Teachings of Unitology"
-					J = "Vested"
-				if("zakarum")
-					B.name = "The Visions of Akarat"
-				if("ianism")
-					B.name = "The Poky Little Puppy"
-				if("adminism", "admintology")
-					B.name = "Breaking Through the Fourth Wall"
-				if("42")
-					B.name = "The Hitchhiker's Guide to the Galaxy"
+		var/new_religion = copytext(sanitize(input(H, "You are the crew's Religious Services Chaplain. What religion do you follow and teach? (Please ensure your ID is in your ID slot before confirming)", "Name of Religion", religion_name)), 1, MAX_NAME_LEN)
+
+		if(!new_religion)
+			new_religion = religion_name //Give them the default one
+
+		var/datum/job/J = H.mind.role_alt_title

```

Comment:
Actually, this is used by the job datum, that's the alternative name of the job, so that should be correct